lib/obsolete.gd

```diff
@@ -479,6 +479,30 @@ DeclareObsoleteSynonym( "MutableIdentityMat", "IdentityMat", "4.8" );
 ##
 DeclareObsoleteSynonym( "MutableNullMat", "NullMat", "4.8" );
 
+#############################################################################
+##
+#F  IsSemilatticeAsSemigroup( <S> ) is the semigroup <S> a semilattice
+##
+##  <#GAPDoc Label="IsSemilatticeAsSemigroup">
+##  <ManSection>
+##  <Prop Name="IsSemilatticeAsSemigroup" Arg='S'/>
+##
+##  <Description>
+##    <C>IsSemilatticeAsSemigroup</C> returns <K>true</K> if the semigroup
+##    <A>S</A> is a semilattice and <K>false</K> if it is not. <P/>
+##
+##    A semigroup is a <E>semilattice</E> if it is commutative and every
+##    element is an idempotent. The idempotents of an inverse semigroup form a
+##    semilattice.
+##
+##    This is identical to <Ref Prop="IsSemilattice" BookName = "Semigroups"/> #
+##    and is present in &GAP;&nbsp;4.8 #  only for the sake of compatibility with
+##    beta-releases.  #  It should <E>not</E> be used in new code.
+##  </Description>
+##  </ManSection>
+##  <#/GAPDoc>
+##
+DeclareObsoleteSynonym( "IsSemilatticeAsSemigroup", "IsSemilattice", "4.8" );
```

Comment:
Is this really right? When I do:

    gap> S := Semigroup(IdentityTransformation);
    <trivial transformation group of degree 0 with 1 generator>
    gap> IsSemilattice(S);
    true
    gap> IsSemilatticeAsSemigroup(S);
    true

No problems, so what is broken by using `DeclareObsoleteSynonym`? 

If it is really necessary to change `DeclareObsoleteSynonym`, then I prefer `DeclareObsoleteSynonymAttr`. I guess there is not much code using this (in particular, there are no methods for `IsSemilattice` in the GAP library except some implications, i.e. true methods).